tests/serialize/runstate/dynamodb_state_store_test.py

```diff
@@ -345,33 +201,32 @@ def test_retry_saving(
     )
     def test_calculate_backoff_delay(self, store, attempt, expected_delay):
         delay = store._calculate_backoff_delay(attempt)
-        assert_equal(delay, expected_delay)
+        assert delay == expected_delay
 
-    def test_retry_reading(self, store):
-        unprocessed_value = {
+    def test_retry_reading(self, store, table_name):
+        """Verify that reading from DynamoDB retries when there are unprocessed keys."""
+        key = store.build_key("job_state", 0)
+        unprocessed_keys_response = {
             "Responses": {},
             "UnprocessedKeys": {
-                store.name: {
-                    "Keys": [{"key": {"S": store.build_key("job_state", 0)}, "index": {"N": "0"}}],
+                table_name: {
```

Comment:
it's probably more future-proof to keep using `store.name` (and would remove a fixture from this test) - but this isn't the end of the world either :p